ruby/example_code/s3/scenario_getting_started.rb

```diff
@@ -47,30 +47,21 @@ def create_bucket
   # Requests a file name from the user.
   #
   # @return The name of the file.
-  def request_file_name
-    file_name = ""
-    while file_name.empty?
-      puts("\nEnter a file you want to upload to your bucket: ")
-      file_name = gets.chomp
-      unless File.file?(file_name)
-        puts("Couldn't find file #{file_name}. Are you sure it exists?")
-        file_name = ""
-      end
-    end
-    file_name
+  def create_file
+    File.open("demo.txt", w) { |f| f.write("This is a demo file.") }
   end
 
-  # Uploads a file from your computer to an Amazon S3 bucket.
+  # Uploads the top-level README from this repo to an Amazon S3 bucket.
```

Comment:
It looks like you are uploading demo.txt instead of the README.

Reply:
Woops. Done.